empire/server/heroku/processes.go

```diff
@@ -100,15 +100,18 @@ type DeleteProcesses struct {
 
 func (h *DeleteProcesses) ServeHTTPContext(ctx context.Context, w http.ResponseWriter, r *http.Request) error {
 	vars := httpx.Vars(ctx)
-	ptype := empire.ProcessType(vars["ptype"])
 	pid := vars["pid"]
 
+	if vars["ptype"] != "" {
+		return errNotImplemented("Restarting a process type is currently not implemented.")
```

Comment:
So, technically we could support this but just submitting a single process instead of all of the processes (just updating a single ECS service).